docs/keyconcepts.rst

```diff
@@ -0,0 +1,89 @@
+.. _keyconcepts:
+
+Key Terms used in Oppia
+========================
+
+.. admonition:: Terms
+
+   * :ref:`Topic<topic>`
+   * :ref:`Story<story>`
+   * :ref:`Skill<skill>`
+   * :ref:`Subtopic<subtopic>`
+   * :ref:`Exploration<explorations>`
+   * :ref:`Card<card>`
+   * :ref:`Content<content>`
+   * :ref:`Interaction<interaction>`
+
+.. _topic:
+
+**Topic:** A topic refers to the subject matter to be taught (e.g., Decimals, Fractions). A topic will consist of a set of stories. 
+
+.. _story:
+
+**Story:** Stories are non-contrived (realistic) scenarios that teach the topic. If the topic is Decimals, then the first story - aiming to introduce the concept - might be a scenario where a customer is in a shop and has to pay an amount in dollars and cents. A story consists of a set of :ref:`explorations <explorations>` or lessons.
+
+.. note:: 
+   Stories within a topic have to be linear, i.e., a student must complete a story before beginning the next story.
+
+.. _skill:
+
+**Skill:** A skill is simply what the student should be able to do. Following our example of Decimals, skills might include 'Comparing Decimals', 'Adding Decimals', 'Converting Decimals to Fractions' etc. Skills are normally associated with a topic.
+
+
+This diagram summarizes the relationship between the above concepts:
+
+.. figure:: /images/Topic_hierarchy.png
+   :alt: Topic hierarchy
+   :align: center
+   
+   *Topic hierarchy in Oppia*
+
+.. _subtopic:
+
+**Subtopic:** In Oppia, a subtopic refers to a group of skills within a topic. Subtopics are created to provide students the opportunity to review and practice skills. 
+
+.. note:: 
+   * If a skill is not assigned to a subtopic, students will not be able to practice that skill. 
+   * If a subtopic is not created for a topic, students will not be able to practice the topic.
```

Comment:
Not sure what this sentence means...

Reply:
Do you mean it's confusing with the way I worded it? Or the information itself is incorrect? (I can reword it if it's a case of it sounding confusing.)
I got this from the **Oppia Structure Project** doc again: 

_Not all skills must be assigned to a subtopic, but this will surface a warning if not the case, as learners will not be able to practice skills not associated with a subtopic. If no subtopics are present for a topic, then learners will not have the ability to practice the topic._

Reply:
Ah, it is technically correct, but the parallelism with the bullet point above it is a little confusing since the meaning of the second word, "a", changes between the two. I suggest rewording the second bullet point as "If no subtopics are created for a topic" or "If a topic contains no subtopics".

Reply:
Done